tests/common/plugins/custom_markers/__init__.py

```diff
@@ -12,6 +16,10 @@ def pytest_addoption(parser):
                          help="only run tests matching the connection CONN_TYPE ('fabric', 'direct')")
         parser.addoption("--device_type", action="store", metavar="DEV_TYPE",
                          help="only run tests matching the device DEV_TYPE ('physical', 'vs')")
+        parser.addoption("--completeness_level", metavar="TEST_LEVEL", action="store", type=int, \
+                         help="Coverage level of test - partial to full execution.\n Defined levels: \
+                         Debug = 0, Basic = 1, Confident = 2, Thorough = 3")
```

Comment:
Define both name and value are confusing, please stick with named levels.

Reply:
Agreed, it will be much better if just named levels are used. Fixed this.